src/app/core/app.component.html

```diff
@@ -59,3 +59,16 @@
   <p i18n="@@auth-error-modal-context">Oops, we are unable to make the site work properly. Are you connected to the Internet?</p>
   <p i18n="@@auth-error-modal-todo">Try to refresh the page. If this problem persists, please contact us.</p>
 </p-dialog>
+<p-dialog
+  *ngIf="hasLanguageError"
+  i18n-header header="Configuration error"
+  [visible]="true"
+  [style]="{width: '50vw'}"
+  [modal]="true"
+  [closeOnEscape]="false"
+  [closable]="false"
+>
+  <p i18n>
+    Language unknown, try <a href="/">reloading the application</a>. If the problem persist, please contact us.
```

Comment:
At first I thought using the fatal error as well, then I tested how the app behaved for [dev.algorea.com](dev.algorea.com) and considered that it was the way the production worked too. Assuming that, the behavior is:

Case 1: if the language is _defined_ but not found, it means the app is hosted for a language that is not in the configured ones. That is a weird case of a devops error, only the administrator(s) of the platform would know what to do.
Reloading with `"/"` would lead to `dev.algorea.com` which will then redirect `dev.algorea.com/en`, providing a nice fallback for the user who will at least be able to access the app. He will then be able to change the language if needed. However, this case 1 scenario is very very unlikely to happen.

Case 2: the language is not _defined_ (when `pathname === '/'`), here the server already falls back to `/en`

Bottom note: That was for the production behavior only

Reply:
Fixed